crates/sui-graphql-rpc/src/error.rs

```diff
@@ -14,8 +14,9 @@ use crate::context_data::db_data_provider::DbValidationError;
 pub(crate) mod code {
     pub const BAD_REQUEST: &str = "BAD_REQUEST";
     pub const BAD_USER_INPUT: &str = "BAD_USER_INPUT";
-    pub const GRAPHQL_VALIDATION_FAILED: &str = "GRAPHQL_VALIDATION_FAILED";
+    pub const _GRAPHQL_VALIDATION_FAILED: &str = "GRAPHQL_VALIDATION_FAILED";
```

Comment:
Changed the error code when logging `query.len > max allowed` to `BAD_USER_INPUT`, thus this is not needed, but I kept it for now.